### DISCUSSION TITLE
Should we also export AVOCADO_TEST_OUTPUT_DIR on runnable-run?

### DISCUSSION THREAD

**Discussion by beraldoleal** [avocado-framework/avocado#3804] [RFC]
After https://github.com/avocado-framework/avocado/pull/3724/commits/7b311fbb33cd253eeb83c6d580ca7a09dd01f785, tests can access an environment variable (`AVOCADO_TEST_OUTPUT_DIR`) to know where to put OUTPUT files.

Today, this is only visible on Task level:

```bash
$ avocado-runner-exec task-run -k exec -u /bin/env -i TASK_ID | grep AVOCADO_TEST_OUTPUT_DIR
```

But when running with `runnable-run` it is not available:

```bash
$ avocado-runner-exec runnable-run -k exec -u /bin/env | grep OUTPUT_AVOCADO_TEST_OUTPUT_DIR
```

Should we change this behavior?

**Comment by willianrampazzo** [comment]
If we consider the following documentation and discussions about the N(ext)Runner:

- https://avocado-framework.readthedocs.io/en/79.0/future/core/nrunner.html
- https://www.redhat.com/archives/avocado-devel/2020-May/msg00009.html
- https://www.redhat.com/archives/avocado-devel/2020-May/msg00015.html

It is clear to me that the only entity responsible for running a test is a Task. In this case, in my opinion, besides testing during the development, I don't see the need to have a `runnable-run` command line. The `task-run` should be the only command-line available to run a test officially.

Based on the previous statement, for the problem described here, the task should be responsible for providing the information on `AVOCADO_TEST_OUTPUT_DIR` to its entities.
